pkg/provisioners/certcentral.go

```diff
@@ -105,7 +107,8 @@ func New(issuer *v1beta1.DigicertIssuer, apiToken string) (*CertCentral, error)
 	return &CertCentral{
 		name:                        fmt.Sprintf("%s/%s", issuer.GetName(), issuer.GetNamespace()),
 		client:                      client,
-		validityYears:               validityYears,
+		validityYears:               &validityYears,
```

Comment:
`validityYears` is set to `defaultValidityYears` above in code. Does it make sense to have a logic below?

```go
        var validityYears *int
        var validityDays *int
        if issuer.Spec.Provisioner.ValidityYears != nil && issuer.Spec.Provisioner.ValidityDays == nil {
                validityYears = issuer.Spec.Provisioner.ValidityYears
        } else {
                validityDays = issuer.Spec.Provisioner.ValidityDays
        }

        if validityYears == nil && validityDays == nil {
                validityYears = &defaultValidityYears
        }
```